vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -121,67 +118,58 @@ spec:
   entandoPluginNamespace: entando
 ```
 
-Now add this link to your environment
+1. Add this link to your environment
 ```
 kubectl create -f my-link.yaml -n entando
 ```
 
-A new link deployer will start and will work behind the scenes to add your plugin `ingressPath` (this is part of the
-plugin `spec`) to the EntandoApp ingress.
+A new link deployer will start and work behind the scenes to add your plugin `ingressPath` (part of the plugin `spec`) to the EntandoApp ingress.
 
-## 5. Upload the micro frontends to your Entando App
+## Upload the Static Resources 
 
-Now that the plugin and the app are linked together, you can proceed to generate the Micro Frontend from the App Builder
+Now that the plugin and app are linked, generate the Micro Frontend from the App Builder
 and upload the static resources like `js` and `css` files.
 
-1. From AppBuilder go to `Configuration -> File Browser` and create a new folder inside the `public` folder and make the name of the folder the same as the name of the bundle (the value is in the `code` field available in the `/bundle/descriptor.yaml` file of the blueprint project) or using a custom name, e.g. `demo-widget`
+1. From the AppBuilder go to `Configuration -> File Browser` 
+2. Migrate to the `public` folder and create a new folder with the same name as the bundle (the value is in the `code` field available in the `/bundle/descriptor.yaml` file of the blueprint project) or use a custom name such as `demo-widget`.
 
-> **Warning**
->
-> If you choose to use a custom folder, remember to update the references in the customUI of the widget later
+3. Upload all the resources in the `/bundle/resources` folder of the project into the new folder
 
-2. Upload all the resources available in the `/bundle/resources` folder
-of the blueprint project into the folder you created above in the App Builder file browser.
-> **Warning**
->
-> You can decide to recreate the folder structure to be the same as the one in `/bundle/resources` or not, but you need to update the
-> references in the customUI / configUI of the widget later if you choose a different folder structure.
+::: tip Warning
+You can duplicate the folder name and structure of `/bundle/resources`, but if you do not, you need to update the references to them in the customUI/configUI of the widget later.
+:::
 
-3. Create the widget. In the App Builder go to `Components → Micro frontends & Widgets`.
+4. In the App Builder go to `Components → Micro frontends & Widgets`  
 
-4. Select `Add`
+5. Select `Add` to create the widget
 
-5. Set whatever title you want
+6. Enter the following with your choices for `anyname`:
+- `Title: anyname1` → enter both English and Italian languages
```

Comment:
enter in both ?

Reply:
Enter in both sounds like you have to enter in the Italian language. Enter for both may work best